app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/ConstraintBinding.kt

```diff
@@ -0,0 +1,30 @@
+package io.github.droidkaigi.confsched2018.presentation.common.binding
+
+import android.databinding.BindingAdapter
+import android.support.constraint.ConstraintLayout
+import android.support.constraint.ConstraintSet
+import android.view.View
+
+@BindingAdapter("layout_constraintDimensionRatio")
+fun View.layout_constraintDimensionRatio(newDimensionRatio: String) {
+    val id = id.takeIf { it != View.NO_ID } ?: let {
```

Comment:
Why do you use let? I think this is simple.
val id = id.takeIf { it != View.NO_ID } ?: return